### PR TITLE
Fixes #22259 : GlassFish docs files migration 

### LINKED ISSUE #22259: GlassFish docs files migration
This task involves :
1. Migrating contents of basedocs/ from javaee/glassfish-docs to nucleus/docs under javaee/glassfish.
2. Making necessary packaging changes in the relevant pom files.

### PR DESCRIPTION
**Pull request by pranjal10**
Fixes #22259.

### CODE REVIEW

**Comment by pranjal10** [comment]
@glassfishrobot Run CI tests please

**Comment by pranjal10** [comment]
@glassfishrobot Run CI tests please

File: appserver/connectors/admin/src/main/manpages/org/glassfish/connectors/admin/cli/flush-connection-pool.1 [hunk 1]
```diff
@@ -33,7 +33,7 @@ DESCRIPTION
        module-scoped-resources are specified using the format
        java:app/jdbc/myDataSource or java:module/jdbc/myModuleLevelDataSource.
        This naming scope is defined in the Java EE 6 Specification
```
Comment (yaminikb) [inline appserver/connectors/admin/src/main/manpages/org/glassfish/connectors/admin/cli/flush-connection-pool.1]: EE 6 to EE 8

**Review by yaminikb — CHANGES_REQUESTED** [review]

**Comment by pranjal10** [comment]
@glassfishrobot Run CI tests please

**Review by yaminikb — APPROVED** [review]

**Pull request merged**